apps/meteor/app/livechat/imports/server/rest/departments.ts

```diff
@@ -147,12 +149,87 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'livechat/department/archived',
+	{
+		authRequired: true,
+		// validateParams: { GET: isGETLivechatDepartmentProps},
+		permissionsRequired: {
+			GET: { permissions: ['view-livechat-departments', 'view-l-room'], operation: 'hasAny' },
+		},
+	},
+	{
+		async get() {
+			const { offset, count } = this.getPaginationItems();
+			const { sort } = this.parseJsonQuery();
+
+			const { text, onlyMyDepartments, excludeDepartmentId } = this.queryParams;
+
+			const { departments, total } = await findArchivedDepartments({
+				userId: this.userId,
+				text,
+				onlyMyDepartments: onlyMyDepartments === 'true',
+				excludeDepartmentId,
+				pagination: {
+					offset,
+					count,
+					sort: sort as any,
+				},
+			});
+
+			return API.v1.success({ departments, count: departments.length, offset, total });
+		},
+	},
+);
+
+API.v1.addRoute(
+	'livechat/department/:_id/archive',
+	{
+		authRequired: true,
+		permissionsRequired: {
+			POST: { permissions: ['manage-livechat-departments'], operation: 'hasAny' },
+		},
+	},
+	{
+		async post() {
+			check(this.urlParams, {
```

Comment:
URL params are "safe" in sense they'll always be a string 🤔 

(If they aren't, for some reason, it would break before reaching here anyways so we can remove the checks for `_id`)